cfgov/preprocessed/js/modules/util/EventObserver.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * EventObserver
+ * @class
+ *
+ * @classdesc Used for creating an object
+ *   that can be used to dispatch and listen to custom events.
+ * @returns {Object} An EventObserver instance.
+ */
+function EventObserver() {
+
```

Comment:
ok, I thought you might want to fire the event on the Dom Element.  How do you access an Expandable instance from a Dom [element](https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/unprocessed/js/modules/beta-banner-state.js#L29)?

On a side-note, OAH has created something similar: https://github.com/cfpb/owning-a-home/blob/5567cf4d89c9ab65f327a98589ba9fd7dd93b2ee/src/static/js/modules/prepare-worksheets/util/event-observer.js.